internal/dca/dca.go

```
@@ -0,0 +1,34 @@
+		log.Errorf("[DCA] Error creating reset-retry-counts request: %s", err.Error())
+		return
+	}
+	req.Header.Set("Content-Type", "application/json")
+
+	client := &http.Client{Timeout: 10 * time.Second}
+	resp, err := client.Do(req)
+	if err != nil {
+		log.Errorf("[DCA] Error calling reset-retry-counts: %s", err.Error())
+		return
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)